sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -199,20 +182,20 @@ public void ThreadCGUD_MemberAUR_MessageGSU_NotificationT()
             //assert
             Assert.AreEqual(updatedTopic, chatThread.Topic);
             Assert.AreEqual(3, chatThread.Members.Count);
-            Assert.AreEqual(messageContent, message.Content);
             Assert.AreEqual(updatedMessageContent, actualUpdateMessage.Value.Content);
-            Assert.AreEqual(ChatMessagePriority.Normal, message.Priority);
+            Assert.AreEqual(ChatMessagePriority.Normal, messagesMapping[1].Item2.Priority);
```

Comment:
we use tuples, we should use named tuples so we do not have to say item1 and item2

Reply:
Witht he next commit, that tuple and dictionary are removed (using a simple array now)